Improve installation instructions (#8454)
grafana/loki: docs/sources/installation/helm/install-scalable/index.md

      Consult the [Reference]({{<relref "../reference">}}) for configuring other storage providers.

>     - In case you're just trying things, you can use the following configuration instead, that sets MinIO as storage:

Reviewer: ```suggestion
    - If you're just trying things, you can use the following configuration instead, that sets MinIO as storage:
```
